javascript-if/main.js

```diff
@@ -0,0 +1,78 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function isEven(number) {
+  if (number % 2) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
This is backwards. Make sure you test your functions 😉

Reply:
<img width="453" alt="Screen Shot 2022-07-09 at 2 27 29 PM" src="https://user-images.githubusercontent.com/97363006/178123478-1080ef42-a810-458b-98bd-8345b50f9751.png">